doc/semantics.md

```diff
@@ -80,6 +80,20 @@ if strict-floats are enabled, or
 
 otherwise.
 
+### `getClass()` for values of primitive types
```

Comment:
You'll have to update the introductory text that says "three exceptions".

Also, if you touch that, it'd be nice to link to a definition of what primitive types are (I assume it's JVM primitive types? or is it IR primitive types? or is it Scala primitive types (is there such a thing))?